### ISSUE TITLE
Not loading posts since Lemmy 0.13.0

### ISSUE THREAD

**Issue by newhoa** [krawieck/lemmur#264]
### Device info

- OS: Android
- OS version: 7.1.2
- Device: Samsung Galaxy
- Lemmur version: 0.6.0

### Describe the bug

When I open Lemmur the post section just shows the spinner. Seems to have started when [Lemmy 0.13.0 was released](https://github.com/LemmyNet/lemmy/releases/tag/0.13.0) . Same thing happened when Lemmy 0.12.0 was released, but Lemmur 0.6.0 fixed that.

Let me know if I can add any additional info. Thanks!

**Comment by shilangyu** [comment]
According to Lemmy's patch notes a simple log out/log in should fix this

**Comment by newhoa** [comment]
When I try to select my profile I get:

`Error: LemmyApiException: not_logged_in`

Long pressing on my account, deleting it, and adding it again seems to have fixed it. Thanks for the tip!

Is this something that happens each time Lemmy updates? If so it's a little confusing.

**Comment by shilangyu** [comment]
> Is this something that happens each time Lemmy updates?

No it is not. As mentioned in [Lemmy 0.13.0 release notes](https://github.com/LemmyNet/lemmy/blob/main/RELEASES.md#major-changes):

> Note: this will log out all users, so users will have to log in again.

This is an exception, it is not expected to happen every time. Lemmur has no control over it. 

However:

> Same thing happened when Lemmy 0.12.0 was released, but Lemmur 0.6.0 fixed that.

This was because the API was changed and it required a new lemmur version to be released. In fact the same happens to the Lemmy web interface, but their releases are much more seamless since it's a webpage. Thus you won't experience these problems there. Lemmy devs are very nice to inform us about those breaking changes before hand so we can release a compatible lemmur version ASAP.

I have created a new issue targeting our unclear way of communication about invalid auth: #265 

Let me know if there is anything else I can help with :)
